src/main/java/no/nav/pto/veilarbfilter/service/OverblikkVisningService.java

```diff
@@ -0,0 +1,36 @@
+package no.nav.pto.veilarbfilter.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.pto.veilarbfilter.domene.OverblikkVisning;
+import no.nav.pto.veilarbfilter.domene.value.VeilederId;
+import no.nav.pto.veilarbfilter.repository.OverblikkVisningRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class OverblikkVisningService {
+
+    private final OverblikkVisningRepository overblikkVisningRepository;
+
+    public Optional<OverblikkVisning> hentOverblikkVisning(VeilederId veilederId) {
+        return overblikkVisningRepository.hent(veilederId);
+    }
+
+    public void lagreOverblikkVisning(VeilederId veilederId, List<String> overblikkVisning) {
```

Comment:
maybe oppdater, and oprett can be one function...if row for veileder already exist then we can update values, or else insert new values...

Reply:
existing flow is also good as well

Reply:
I did it that way at first (single method), but then I decided to split it up after all. I guess I thought that doing it simple (i.e. no "smartness") made it easier to reason about the code during my testing.

Reply:
Valid logic